### DISCUSSION TITLE
How do I build the installer to install OKD rather than OCP

### DISCUSSION THREAD

**Discussion by fortinj66** [okd-project/okd#1183] [Q&A] [answered]
I'm missing something fundamental. 

I'm patching openshift-installer and I can build it fine.  However, when I go to use it, it defaults to OCP and I get RHCOS images instead of FCOS images.  How do I tell the installer to build OKD rather than OCP.  Or do I have to add the new installer image to an OKD release and then do `oc adm release extract ...`

**Comment by vrutkovs** [comment, marked as answer]
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/installer/openshift-installer-master__okd.yaml#L101-L103 - you need to pass `--build-args TAGS=okd` when building it in container or `export TAGS=okd` when running `hack/build.sh`

**Reply by fortinj66** [reply]
Thanks!  That's what I needed...

**Reply by fortinj66** [reply]
Works for `installer`. 

Only seems to work for the first build on `installer-artifacts`

**Reply by fortinj66** [reply]
https://github.com/openshift/installer/pull/5794 fixes the installer-artifacts issue
